google-cloud-firestore/src/test/java/com/google/cloud/firestore/MapperTest.java

```diff
@@ -972,6 +972,15 @@ private static void assertExceptionContains(String partialMessage, Runnable run)
     }
   }
 
+  private static void assertExceptionThrows(Runnable run) {
+    try {
+      run.run();
+      fail("Expected exception not thrown");
+    } catch (RuntimeException ignored) {
```

Comment:
This should be a more specific exception

Reply:
Will do